cluster-autoscaler/FAQ.md

```diff
@@ -148,6 +160,21 @@ All in all the total reaction time is around 4 min.
 
 # Troubleshooting:
 
+### I have couple nodes with low utilization but they are not scaled down. Why?
+
+CA doesn't remove nodes if they are running system pods, pods without a controller or pods with local storage. 
+Also it won't remove a node which has pods that cannot be run elsewhere due to limited resources. Another option
+is that the corresponding node group already has the minimum size. CA doesn't scale down if there was a scale up
+in the last 10 min.
+Please check if this is the case with your cluster.
```

Comment:
Does this sentence only refer to last point (scale up in last 10 min)? That's how I would interpret it grammatically, but if that's the case the formatting is very confusing. Otherwise I think it should be something like "Please check if any of the above apply to your cluster" or something like that.